alpha/apps/kaltura/lib/batch2/kFlowHelper.php

```diff
@@ -251,6 +256,24 @@ public static function handleImportFinished(BatchJob $dbBatchJob, kImportJobData
 		return $dbBatchJob;
 	}
 
+	private function excuteClip($flavorAsset, $dbBatchJob) {
+		$entryFlavors = assetPeer::retrieveFlavorsByEntryId($flavorAsset->getEntryId());
+		$originalFlavorAsset = assetPeer::retrieveOriginalByEntryId($flavorAsset->getEntryId());
+		foreach($entryFlavors as $entryFlavor) {
+			$flavor = assetParamsOutputPeer::retrieveByAsset($entryFlavor);
+			kBusinessPreConvertDL::decideFlavorConvert($entryFlavor, $flavor, $originalFlavorAsset, null, null, $dbBatchJob);
```

Comment:
Avoid using decideFlavorConvert. 
Do the following - 1 . Get source info. 2. Add Convert job on source.